go/vt/vtorc/inst/instance_dao.go

```diff
@@ -711,38 +692,17 @@ func ReadInstancesWithErrantGTIds(keyspace string, shard string) ([]*Instance, e
 	return readInstancesByCondition(condition, args, "")
 }
 
-// GetKeyspaceShardName gets the keyspace shard name for the given instance key
-func GetKeyspaceShardName(tabletAlias string) (keyspace string, shard string, err error) {
```

Comment:
`inst.ReadTablet` will get you `keyspace`/`shard` from the same table, so let's just use that 🤷